internal/sessiontest/session_test.go

```diff
@@ -28,7 +29,15 @@ func TestSigningSession(t *testing.T) {
 func TestDisclosureSession(t *testing.T) {
 	id := irma.NewAttributeTypeIdentifier("irma-demo.RU.studentCard.studentID")
 	request := getDisclosureRequest(id)
-	sessionHelper(t, request, "verification", nil)
+	responseString := sessionHelper(t, request, "verification", nil)
+
+	// Check default JWT validity
+	claims := jwt.MapClaims{}
+	jwt.ParseWithClaims(responseString, claims, nil)
+
+	iat := claims["iat"].(float64)
+	exp := claims["exp"].(float64)
+	require.True(t, iat+120000 == exp)
```

Comment:
- If the JWT would put something else than floats in here, then these type assertions would panic. You can write `exp, ok := claims["exp"].(float64)` and then check `ok` first to check that we indeed received a float (and fail the test if not).
- Using equality checks on floats is generally not a good idea because rounding errors may bite you.

In this case, we can avoid both issues because we know exactly the structure of the data that should be inside the JWT. The following should work:
```go
claims := struct {
	jwt.StandardClaims
	*server.SessionResult
}
_, err = jwt.ParseWithClaims(responseString, claims, nil)
```
Then you can refer to the `iat` and `exp` fields by `claims.IssuedAt` and `claims.ExpiresAt` and receive `int64`'s (and as an extra, you could require the presence of some of the other expected fields).

Reply:
Pushed new commit with custom claims resulting in ints as you suggested.